topics/client-create-new-application.topic

```diff
@@ -244,9 +244,31 @@
                 <code-block
                         lang="kotlin"
                         src="snippets/tutorial-client-get-started/src/main/kotlin/Main.kt"/>
+            </step>
+            <step>
                 <p>
-                    Proceed to the next section to learn how to run your application.
+                    To see logs correctly, you need to add a logger dependency. In the
```

Comment:
Should we explain how the logging works and about SLF4J?

Reply:
I've added more information in the relevant topics. (preview links in description) 
Can you please verify?